deploy/helm/secret-operator/templates/daemonset.yaml

```diff
@@ -56,6 +56,8 @@ spec:
         - name: external-provisioner
           image: "{{ .Values.csiProvisioner.image.repository }}:{{ .Values.csiProvisioner.image.tag }}"
           imagePullPolicy: {{ .Values.csiProvisioner.image.pullPolicy }}
+          resources:
+            { { .Values.node.resources | toYaml | nindent 12 } }
```

Comment:
WDYT of putting external-provisioner and node-driver-registrar resources behind a dedicated config option?
i think we should make them configurable indivudally

Reply:
Ok, this comment also infects listener-op

Reply:
I'll defer to @nightkr on that

Reply:
Good catch, :+1: on that

Reply:
ok, I'll keep my hands of this for now then, correct?

Reply:
Ah sorry, :+1: as in "yes this sounds like a good idea to do".

Reply:
ok, I'll do that